src/main/java/com/feelfreecincinncati/dto/EventDTO.java

```diff
@@ -23,18 +23,24 @@ public String getAddress() {
 		return Address;
 	}
 	public void setAddress(String address) {
-		Address = address;
+		this.Address = address;

```

Comment:
Make this lowercase, while you're here.  In Java, variables are lowerCamelCase, not UpperCamelCase.